server/webhook.go

```diff
@@ -970,59 +996,118 @@ func (s *Server) prepareIssueFromSDLFile(ctx context.Context, repo *api.Reposito
 	return migrationDetailList, nil
 }
 
+type migrationFileYAMLDatabase struct {
```

Comment:
Consider moving this to api package since 1) this is API, 2) we can give some examples about it, and 3) test package can use this API.